<?xml version="1.0"?>
<launch>

  <arg name="Name" default="data.csv"/>

  <node name="PID_output_plotter" pkg="two_wheel" type="PID_csv_outputter.py" output="screen">
    <param name="csv_path" value="$(find two_wheel)/csv/$(arg Name)"/>]
  </node>
  <node name="Arduino_node" pkg="rosserial_python" type="serial_node.py">
    <param name="port" value="/dev/ttyS1"/>
  </node>


</launch>
